noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -1433,6 +1441,7 @@
                     FROM arkivstruktur.dokumentbeskrivelse d
                     JOIN arkivstruktur.registrering r ON r.systemid = d._ref_registrering
```

Comment:
The guideline on which we based the extraction validator most of the tests were implemented with regard to the series in which the objects reside.

Considering that, the two `JOINs` are there to extract the series system ID and the `LEFT JOIN` is there to extract information about the parent File ID (if any).